src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -244,27 +169,13 @@ public class DurableTaskOptions
         /// <value>Assembly qualified class name that implements <see cref="ILifeCycleNotificationHelper">ILifeCycleNotificationHelper</see>.</value>
         public string CustomLifeCycleNotificationHelperType { get; set; }
 
-        /// <summary>
-        /// Gets or sets the maximum queue polling interval.
-        /// </summary>
-        /// <value>Maximum interval for polling control and work-item queues.</value>
-        public TimeSpan MaxQueuePollingInterval { get; set; } = TimeSpan.FromSeconds(30);
-
         // Used for mocking the lifecycle notification helper.
         internal HttpMessageHandler NotificationHandler { get; set; }
 
         internal string GetDebugString()
         {
             var sb = new StringBuilder(4096);
             sb.AppendLine("Initializing extension with the following settings:");
```

Comment:
What is the plan to get the other Options types to participate in populating this debug string?

Reply:
I think I have addressed this now.